### DISCUSSION TITLE
feature request: Allow regex in abbreviations

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6706] [Ideas]
hey it'd be cool if you could put regex within abbreviation terms

**Comment by kamilkrzyskow** [comment]
[Abbreviations](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#abbreviations) is an external markdown extension, the theme only adds styling to the processed result, so this should be an upstream request to either [Python Markdown](https://github.com/Python-Markdown/markdown) or [Python Markdown Extensions](https://github.com/facelessuser/pymdown-extensions) or you can modify the [abbr](https://github.com/Python-Markdown/markdown/blob/master/markdown/extensions/abbr.py) extension yourself.

**Reply by couleurm** [reply]
sorry about #6842 i genuinely forgot i already made this
